src/lib/components/event/event-group-details.svelte

```diff
@@ -7,10 +10,27 @@
     eventOrGroupIsTerminated,
   } from '$lib/models/event-groups/get-event-in-group';
   import { isLocalActivityMarkerEvent } from '$lib/utilities/is-event-type';
+  import { routeForEventHistory } from '$lib/utilities/route-for';
 
   export let eventGroup: EventGroup;
   export let selectedId: string;
   export let onGroupClick: (id: string) => void;
+
+  const handleGroupClick = (id: string) => {
+    const { namespace, workflow, run } = $page.params;
```

Comment:
We can probably just store these variables in the top-level instead of fetching them every time.